.travis.yml

```diff
@@ -8,19 +8,18 @@ cache:
 install:
   - pip3 install -r requirements.txt
 
-script:
-  - py.test -v --color=yes --cov=.
+jobs:
+  include:
+    - stage: test
+      script: py.test -v --color=yes --cov=.
+    - stage: deploy
+      skip_cleanup: true
+      provider: script
+      script: bash etc/deploy_docs.sh
 
 after_success:
```

Comment:
Putting the codecov outside of build stages is making it run at the end of both the build stages. It is better to put it in **deploy** stage for now.